Model/Config.php

```diff
@@ -142,6 +143,22 @@ public function convertHistoricalCarts($store = null): bool
         );
     }
 
+    /**
+     * Should set the cart's customer to null when `Reclaim`ing an abandoned cart
```

Comment:
```suggestion
     * Config flag for whether the abandoned cart reclaiming should disassociate any previous Magento customers from the quote.
```

Nit: the comment didn't quite communicate what I expected. Not sure if my suggestion is any better though 😄